frontend/test/__support__/e2e/cypress.json

```diff
@@ -9,6 +9,14 @@
   "projectId": "KetpiS",
   "viewportHeight": 800,
   "viewportWidth": 1280,
+  "reporter": "mochawesome",
+  "reporterOptions": {
+      "reportDir": "cypress/mochawesome-report",
+      "overwrite": false,
```

Comment:
Not overwriting to be able to get all test results from a scenario (folder, different .specs)